model/main.py

```diff
@@ -0,0 +1,33 @@
+import sys
+from model.model import Model
+
+from argparse import ArgumentParser
+
+
+def setup_parser() -> ArgumentParser:
+    parser = ArgumentParser(description="Read file from Command line.")
+    parser.add_argument(
+        "--aug-airports-load-file",
+        required=True,
+        type=str,
+        help="file path for information about each airport",
+    )
+    parser.add_argument(
+        "--model_dump_dir",
+        required=True,
+        type=str,
+        help="path to directory to store model",
+    )
+    return parser
+
+
+def main() -> None:
+    parser = setup_parser()
+    args = parser.parse_args(sys.argv[1:])
```

Comment:
This seems odd to me - I'd usually expect the main function to accept the list of command line arguments and do something with them instead of accessing the directly, so the definition would be more like

```python
def main(args: list[str]) -> None:
    parser = setup_parser()
    args = parser.parse_args(args)
    ...
```